index.d.ts

```diff
@@ -0,0 +1,93 @@
+import { Test } from 'unist-util-is';
```

Comment:
Could we include typescript files in the prettier formatter in package.json?
So that it matches the code style.

Reply:
Also could we include the typescript version required as a comment?
similar to https://github.com/syntax-tree/hast-util-to-html/blob/0fef5aa542e1d8a9361f3a0ea8e38adf07fa870f/types/index.d.ts#L1